ui/apps/platform/cypress/integration/clusters/clusters.test.js

```diff
@@ -48,6 +49,24 @@ describe('Clusters page', () => {
             });
         });
     });
+
+    describe('when no secured clusters are added yet (only applies to Cloud Service)', () => {
+        it('should should redirect to the Clusters page', () => {
+            visitDashboardWithNoClusters();
+
+            cy.url().should('contain', `${clustersUrl}`);
+
+            cy.get(selectors.clustersListHeading);
+
+            cy.get(
+                'p:contains("You have successfully deployed a Red Hat Advanced Cluster Security platform.")'
+            );
+
+            cy.get('h1:contains("Configure the clusters you want to secure.")');
```

Comment:
Accessibility: how about `h2` element here, because **Clusters** page already has `h1` element?

Reply:
Good point, but tricky because the important information is that "you aren't monitoring any clusters yet".
I'll see if I can make it a semantic h2, but with h1 styling.